frontend/src/metabase/visualizations/components/ChartSettings.jsx

```diff
@@ -26,6 +27,30 @@ import {
 // section names are localized
 const DEFAULT_TAB_PRIORITY = [t`Display`];
 
+const popoverWidget = (widgets, currentWidget, extraWidgetProps) => {
```

Comment:
It still returns an array of 0 or 1 elements, doesn't it? Could we make it return a widget or undefined instead?